addons/udes_stock/models/stock_picking.py

```diff
@@ -508,11 +508,25 @@ def _create_backorder_picking(self, moves, **kwargs):
         to create backorders when pickings are validated regardless of the user's generic
         permissions.
         """
+        multi_users_enabled = self.picking_type_id.u_multi_users_enabled
         # If configuration is enabled, copy the batch to the backorder too.
         batch_id = False
         if self.picking_type_id.u_preserve_backorder_batch:
             # If the picking has no batch, batch_id will be False.
             batch_id = self.batch_id.id
+        # If multi users config is enabled, moves to keep will stay in the existing picking and others will be
```

Comment:
Q: Why? Could we document the reason why we do this?

Reply:
Added some other comments on below lines which hopefully explains what is done